CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [unreleased]
 
 - Fix: Update `:hover` and `:focus` state on `azote` button.
+- Fix: Fix `CrowdfundingCard` marges.
```

Comment:
C'est pas `margins` ?

Reply:
Non y'a aussi un padding dans le tas :).

Reply:
Hum, margin and padding sinon

Reply:
`spacing` du coup xD

Reply:
Parfait !